.eslintrc

```diff
@@ -10,10 +11,10 @@
 	"rules": {
 		"brace-style": ["warn", "stroustrup", { "allowSingleLine": true }],
 		"func-call-spacing": ["warn", "never"],
-		"indent": ["warn", 2],
+		"indent": ["warn", "tab", { "ignoredNodes": ["TemplateLiteral *"] }],
```

Comment:
Made sure this was aligned with our prettier settings so they weren't fighting with each other